open-source-workflow.html

```diff
@@ -99,7 +99,7 @@ <h4 class="requirements">Testing</h4>
 
   <div class="workflowBlush5 workflowChart">
         <h4 class="requirements">Maintenance</h4>
-        <p>Create Issues that Fix Errors/Improve Impelentation/Upgrade the Current Language <i class="fas fa-long-arrow-alt-right"></i> Approved by Mentor/Maintainer/Committer</p>
+        <p>Create Issues that Fix Errors/Improve Impelentation/ Upgrade the Current Language <i class="fas fa-long-arrow-alt-right"></i> Approved by Mentor/ Maintainer/ Committer</p>
```

Comment:
@shreyateeza Since you are changing this paragraph, which includes the incorrect spelling of implementation, please change the spelling as well, and add a new commit for spelling correction.

Reply:
@sara-02 I have created a separate issue for the correction of spelling. Also AnkitKhatri55555 has created a PR for it. So shall I still do the amendments?

Reply:
@shreyateeza No, not needed, we can use her PR to resolve the other issue. :)